WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCToken/VCToken/algorithms/PublicJWK.swift

```diff
@@ -3,8 +3,8 @@
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
 
-/// TODO: deprecate entity for JWK
-public struct ECPublicJwk: Codable, Equatable {
+public struct PublicJWK: Codable, Equatable 
```

Comment:
This is still an ECPublicKey. `x` and `y` are only on keytype EC.

Reply:
Can we just keep the name EcPublicJwk for now and resolve this in a different PR?